packages/bstrings.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'bstrings'
+$category = 'Forensic'
```

Comment:
We have similar tools like floss in `Utilities` instead of `Forensic`. I would add this tool in `Utilities`. Alternatively, we could consider introducing a `Strings` category as we have a few tools related with extracting strings. @vm-packages opinions?

Reply:
I change the category to `Utilities`